mlos_bench/mlos_bench/config/schemas/services/remote/azure/azure-service-subschema.json

```diff
@@ -0,0 +1,66 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/microsoft/MLOS/main/mlos_bench/mlos_bench/config/schemas/services/remote/azure/azure-service-subschema.json",
+    "title": "mlos_bench Azure Base Service config",
+    "description": "Base config schema for an mlos_bench Azure Service",
+    "$defs": {
+        "azure_service_config": {
+            "type": "object",
+            "allOf": [
+                {
+                    "$ref": "../../common-defs-subschemas.json#/$defs/poll_configs"
+                },
+                {
+                    "properties": {
+                        "subscription": {
+                            "description": "Azure subscription id (typically provided in the global config in order to omit from source control).",
+                            "type": "string"
+                        },
+                        "resourceGroup": {
+                            "description": "The name of the resource group to place the deployment in (typically provided in the global config in order to omit from source control).",
+                            "type": "string"
+                        },
+                        "deploymentName": {
+                            "$comment": "TODO: provide templating strings to augment names via experiment_id/trial_id, for instance.",
```

Comment:
- [ ] implement this and remove the comment

Reply:
This might actually need to be moved out of the service config and into the environment, which is a larger change, so I think I'll postpone that until later.

Reply:
Will do this in #610 instead.